src/plugins/vis_type_timeseries/public/application/components/splits/terms.js

```diff
@@ -81,11 +80,7 @@ export const SplitByTermsUI = ({
   const selectedField = find(fields[indexPattern], ({ name }) => name === model.terms_field);
   const selectedFieldType = get(selectedField, 'type');
 
-  if (
-    seriesQuantity &&
-    model.stacked === STACKED_OPTIONS.PERCENT &&
-    isPercentDisabled(seriesQuantity[model.id])
-  ) {
+  if (seriesQuantity && model.stacked === STACKED_OPTIONS.PERCENT && seriesQuantity[model.id] < 2) {
```

Comment:
Nit: Don't you want to move condition `seriesQuantity[model.id] < 2` to separate function in case, if this condition will become more complex, and describe check by function name? It occurs two times in the code.